### PR TITLE
Add Fedora 30 Dockerfiles (based on 29 ones)

### PR DESCRIPTION
**Pull request by MattGal**
@dseefeld FYI

### CODE REVIEW

File: src/fedora/30/amd64/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        libgdiplus \
+        libicu-devel \
+        libunwind-devel \
+        libuuid-devel \
+        lttng-ust-devel \
+        uuid-devel \
+    && dnf clean all
+
+# Until official packages are available, we have to restore the ubuntu ones instead.
+ENV __PUBLISH_RID=ubuntu.14.04-x64
```
Comment (MichaelSimons) [inline src/fedora/30/amd64/Dockerfile]: I've never understood the need for this, can you explain?  If this is needed, should it be updated to something closer to Fedora 30 such as Fedora 29?

Comment (MattGal) [reply src/fedora/30/amd64/Dockerfile]: I don't understand it either, I copied it off the Fedora 29 dockerfiles.  Should I delete it?

Comment (MichaelSimons) [reply src/fedora/30/amd64/Dockerfile]: I vote to delete it until someone needs it in which they can add it and set it accordingly.

Comment (MattGal) [reply src/fedora/30/amd64/Dockerfile]: Done

**Comment by MattGal** [comment]
Thanks Michael! Fedora build already finished but I like green stuff so I'll wait til the others finish to merge...

**Pull request merged**